rasa/utils/common.py

```diff
@@ -75,10 +75,15 @@ def set_log_level(log_level: Optional[int] = None):
 def update_apscheduler_log_level():
     log_level = os.environ.get(ENV_LOG_LEVEL_LIBRARIES, DEFAULT_LOG_LEVEL_LIBRARIES)
 
-    logging.getLogger("apscheduler.scheduler").setLevel(log_level)
-    logging.getLogger("apscheduler.scheduler").propagate = False
-    logging.getLogger("apscheduler.executors.default").setLevel(log_level)
-    logging.getLogger("apscheduler.executors.default").propagate = False
+    apscheduler_loggers = [
+        "apscheduler",
+        "apscheduler.scheduler",
+        "apscheduler.executors.default",
```

Comment:
Since you added `apscheduler` also and If it's needed to set the log level individually for all levels in the package hierarchy, then should we also add `apscheduler.executors`

Reply:
Good question if we need that. I'll add it for consistency.